Contrib/README.MSXPi

```diff
@@ -0,0 +1,162 @@
+on a Python msxpi-server code running on the same computer where 
+openMSX is hosted.
+
+For detailed information about the MSXPi project and the hardware interface,
+go to: https://github.com/costarc/MSXPi
+
+This README is for the openMSX extension that reproduces the behaviour of
+the physical interface.
+
+Architecture of the physical Interface:
+┌────────────┐         ┌────────────────────────┐         ┌────────────────────┐
+│    MSX     │  <--->  │     MSXPi Interface    │  <--->  │   msxpi-server     │
+│ (Z80 I/O)  │   0x56  │   (CPLD + SPI bridge)  │  GPIO   │ (Python GPIO API)  │
+└────────────┘   0x5A  └────────────────────────┘         └────────────────────┘
+                                                 
+Architecture of the MSXPi extension for openMSX: 
+┌────────────┐         ┌────────────────────────┐         ┌────────────────────┐
+│    MSX     │  <--->  │     MSXPi Interface    │  <--->  │   msxpi-server     │
+│ (Z80 I/O)  │  0x56   │ MSXPiDevice + socket)  │  Socket │(Python socket API) │
+└────────────┘  0x5A   └────────────────────────┘         └────────────────────┘
+
+Now that you know everything you need (not really!) to start with MSXPi, prepare
+the environment.
+
+To use this extension, you will need Python3 installed in your computer.
+IDLE will work with MSXPi in Windows - it can be downloaded and installed
+from Microsoft App store. MSXPi will also work in other platforms - it
+works better on Linux, since it is the native environment it was designed
+for initially.
+At this time, some commands will work on Windows, many won't - but they
+are being ported and pushed to MSXPi repository, so I recommend to keep
+refreshing your local copy frequently.
+
+There are a lot of Python modules that are not standard in Python, and will
+require you to install before running MSXPi. When you run msxpi-server, it
+will complain about missing modules - use that information to install them.
+For example, to install module "fs":
+
+Go to Windows command line and type:
+
+   python -m pip install fs
+
+Repeat for all modules that are not already installed.
+
+Some commands (such as ploadr) will try to use 7-Zip, LHA or UNAR to 
+decompress files.. You will need to install those tools also or the
+command will fail.
+
+You will also have to download several files needed by MSXPi - follow the 
+instructions below to get started.
+
+Steps to Setup and Try MSXPi in openMSX
+=======================================
+
+1. Install an openMSX version that has support for MSXPi extension. You can download
+   binaries for all platforms (nightly builds from openMSX) in the releases section:
+   https://github.com/costarc/openMSX/releases
+   
+   Always use the latest release.
+   
+2. Download msxpi-server.py from:
+
+   https://github.com/costarc/MSXPi/tree/master/software/Server/Python/src
+
+
+   Note: the msxpi-server component has some commands that requires elevation of privileges
```

Comment:
Note: The msxpi-server component has some commands that **require** elevated privileges on the Raspberry Pi platform using "sudo".